### ISSUE TITLE
[Guide] How to use environment variables with bundle

### ISSUE THREAD

**Issue by gregurco** [8p/EightPointsGuzzleBundle#220]
Would be nice to have article with example how to use environment variables to configure bundle.
For example, base url can be configured from **.env** file.

**Comment by mboron83** [comment]
Hi. As @gregurco said it would be nice to have oportunity to use environment variables.I tried to do this with example below:
```
# config/packages/eight_points_guzzle.yaml
eight_points_guzzle:
    clients:
        httpbin_client:
            base_url: "%env(HTTPBIN_CLIENT_BASE_URL)%"
```

Unfortunately it gives me an error:

> In BaseNode.php line 422:                                                                                                         
> Invalid configuration for path "eight_points_guzzle.clients.aol_papi.base_url": base_url can be: string

**Comment by gregurco** [comment]
Implemented in #241
